product_tax_multicompany_default/README.rst

```diff
@@ -0,0 +1,98 @@
+=================================
+Product Tax Multi Company Default
+=================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmulti--company-lightgray.png?logo=github
+    :target: https://github.com/OCA/multi-company/tree/12.0/product_tax_multicompany_default
+    :alt: OCA/multi-company
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/multi-company-12-0/multi-company-12-0-product_tax_multicompany_default
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/133/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module sets the default company taxes for all the existing companies when
+a product is created. It also adds a button in product view to set all the
+taxes from other companies matching them by tax code.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+* As obvious, you need to have several companies.
+* Go to Accounting > Configuration > Settings
+* Select the proper company (if you are admin or a user with several companies
+  access), or keep the current one (for a regular user).
+* On Invoicing & Payments section, set fields "Default Sale Tax" and "Default
+  Purchase Tax".
+
+Usage
+=====
+
+* Go to Sales > Sales > Products.
```

Comment:
Why have you changed the numbered list by an unordered one?

Reply:
Because taking a look I saw that fashion was to put asterisks :smile:

Reply:
Well, that's old fashion :stuck_out_tongue_closed_eyes: See template when you have doubts: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/readme/USAGE.rst

Reply:
Changes done.
I like to be always up to date. :smiley: